views/layouts/layout-pane.njk

```diff
@@ -8,16 +8,35 @@
   <div class="app-o-pane__content">
     <main id="content" class="app-c-content" role="main">
       <div class="app-c-content__header">
-        <h1 class="govuk-heading-xl">
+
+        {% if status %}
+        <h1 class="govuk-heading-xl govuk-!-mb-r2">
           <span class="govuk-caption-xl">
           {% if theme %}
-              {{ theme }}
+            {{ theme }}
             {% else %}
               {{ section }}
-          {% endif %}
+            {% endif %}
           </span>
           {{ title }}
+          </h1>
+          <a href="#next-steps">
+            <strong class="govuk-c-tag app-c-tag__status govuk-!-mb-r8">
```

Comment:
This is a _modifier_ (we're modifying the tag to add additional behaviour) so it should be `app-c-tag--status`. Also, I think it'd make more sense to add the class to the link?

Reply:
Fixed